src/openstudio_lib/SchedulesView.cpp

```diff
@@ -613,14 +610,15 @@ ScheduleTab::ScheduleTab(const model::ScheduleRuleset& schedule, SchedulesView*
   : QWidget(parent),
     //m_mouseDown(false),
     m_selected(false),
+    m_header(new ScheduleTabHeader(this)),
```

Comment:
@jmarrec there is an issue here because `ScheduleTabHeader::ScheduleTabHeader` tries to call `SchedulesView::schedule` before `m_schedule` is set.  I fixed that but hit another error.  I think some of these changes to move construction from ctor body to initializer list might not be ok in this file at least.